spec/gameSpec.js

```diff
@@ -0,0 +1,84 @@
+describe('game', function() {
+
+});
+
+  var testGame;
+  var testFrameOne;
+  var testFrameTwo;
+  var testFrameThree;
+  var testFrameFour;
+  var testFrameFive;
+  var testFrameSix;
+  var testFrameSeven;
+  var testFrameEight;
+  var testFrameNine;
+  var testFrameTen;
+  var frames;
+//
+  beforeEach(function() {
+    testGame = new game();
```

Comment:
This feels like a lot of test setup. We're duplicating a lot of effort by instantiating individual frames and assigning individual scores. That's usually a sign that a classes dependencies are too tightly coupled.

If we think about this in a TDD fashion, out tests should look a lot simpler. For example, if we want to test some logic to calculate a frame bonus score then the things we should care about are (a) we have a frame that should get a bonus score (the condition) (b) we have a roll which will generate that bonus score (the action) and (c) an expectation of what that bonus score should be (the result). 

To achieve a really clean test, the only dependency that we should need to inject into our game is a frame that requires a bonus score.